trezorlib/tests/device_tests/test_msg_stellar_sign_transaction.py

```diff
@@ -37,15 +39,35 @@ def get_address_n(self):
     def test_sign_tx_bump_sequence_op(self):
         self.setup_mnemonic_nopin_nopassphrase()
 
-        xdr = b64decode("AAAAABXWSL/k028ZbPtXNf/YylTNS4Iz90PyJEnefPMBzbRpAAAAZAAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAt//////////wAAAAAAAAAA")
+        op = proto.StellarBumpSequenceOp()
+        op.bump_to = 9223372036854775807
```

Comment:
Can we write this as 0x7fffffffffffffff ? (It's easier to see that this value is valid).

Reply:
Definitely, I haven't realized this.